### ISSUE TITLE
How to send the DB user in your FB example

### ISSUE THREAD

**Issue by piyushbeli** [enhancv/popup-tools#1]
Thanks for the nice tool you created for all purpose popup usages.

I have a question regarding your FB Passport example. In below code statement it returns the raw FB user but I want to return the serialized user (which is the user from DB) to the client. Can you help me to achieve that?

`
app.get('/callback-url',
    passport.authenticate('facebook'),
    function (req, res) {
        res.end(popupTools.popupResponse(req.user))
    }
);
`

so basically I want the user from here
`
PASSPORT.serializeUser(function(req, user, cb)  {})
`

**Comment by piyushbeli** [comment]
Never mind. I found a way to access the passport user. At this point, we can access it by req.session.passport.user
